wicket-extensions/src/main/java/org/apache/wicket/extensions/ajax/markup/html/autocomplete/wicket-autocomplete.js

```diff
@@ -183,6 +184,12 @@
 
 							hideAutoComplete();
 
+							if (cfg.focusInputOnTabSelection && keyCode === KEY_TAB) {
```

Comment:
If you like my idea of using an enum, then here the code would look like:
```suggestion
							if (cfg.focusInputOnTabSelection == "some_enum_value" && keyCode === KEY_TAB) {
```